packages/deployments/contracts/contracts/messaging/libraries/Queue.sol

```diff
@@ -121,6 +131,14 @@ library QueueLib {
     return items;
   }
 
+  /**
+   * @notice Sets a certain value to be ignored (skipped) when dequeuing.
+   */
+  function remove(Queue storage queue, bytes32 item) internal {
+    require(!queue.removed[item], "already removed");
+    queue.removed[item] = true;
+  }
```

Comment:
Seems like fix for RootManager.sol who will call this function is missing in this pull request. Kindly check